packages/cli/src/controller/init-controller.ts

```diff
@@ -15,7 +15,7 @@ const STARTER_PATH = 'https://github.com/subquery/subql-starter';
 export async function createProject(localPath: string, project: ProjectSpec): Promise<void> {
   const projectPath = path.join(localPath, project.name);
   try {
-    await simpleGit().clone(STARTER_PATH, projectPath);
+    await simpleGit().clone(STARTER_PATH, projectPath, ['-b', 'v0.0.2', '--single-branch']);
```

Comment:
TODO: Remove once https://github.com/subquery/subql-starter/pull/30

Reply:
Actually its probably a good idea to never merge this branch, CLI should always point to a specific version. Merging this would be break `init` for any CLI version before this.

Reply:
I'm thinking we leave `init` untouched. So even with this PR merged, end user still by default init a v0.0.1 project.

Reply:
Done. We also might want to offer a CLI command to update projects from 0.0.1 to 0.2.0